packages/ember-testing/lib/helpers/click.js

```diff
@@ -16,22 +16,34 @@ import { focus, fireEvent } from '../events';
   });
   ```
 
+  Advanced example, overriding the client coordinate and using the right mouse button:
+
+  ```javascript
+  click('.some-jQuery-selector', null, {button: 3, clientX: 17, clientY: 40}).then(function() {
+    // assert something
+  });
+  ```
+
   @method click
-  @param {String} selector jQuery selector for finding element on the DOM
-  @param {Object} context A DOM Element, Document, or jQuery to use as context
+  @param {String} selector jQuery selector for finding an element in the DOM. Note that only the first match will be clicked.
+  @param {Object} context A DOM Element, Document, or jQuery to use as context.
+    If null or undefined, default is the root application DOM element.
+  @param {Object} event An optional hash of event parameters to mock the JQuery [Event]{@link http://api.jquery.com/category/events/event-object/} object.
```

Comment:
this may be considered a new feature, although I could be convinced this is a bugfix

@rwjblue what do you think good sir.